packages/faceted-search/src/components/Badges/BadgeNumber/BadgeNumberForm.component.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from 'enzyme';
+import { BadgeNumberForm } from './BadgeNumberForm.component';
+import { BadgeFacetedProvider } from '../../context/badgeFaceted.context';
+
+const badgeFacetedContextValue = {
+	onDeleteBadge: jest.fn(),
+	onHideOperator: jest.fn(),
+	onSubmitBadge: jest.fn(),
+};
+
+describe('BadgeTextForm', () => {
+	it('should mount a default badge', () => {
+		// Given
+		const props = {
+			id: 'customId',
+			onSubmit: jest.fn(),
+			t: () => 'Apply',
```

Comment:
This is a bad thing I've done, it will be better to have a proper translate function.
The mounted component will not be relevant with all t returning an 'apply'.

Reply:
Just change to a better mock function, this also update snapshots with better translation strings